src/utils/optionsdict.cc

```diff
@@ -77,6 +77,24 @@ bool OptionsDict::HasSubdict(const std::string& name) const {
   return subdicts_.find(name) != subdicts_.end();
 }
 
+template <>
+bool OptionsDict::GetOnce(const std::string& key) const {
+  for (const auto* alias : aliases_) {
+    const auto& dict = alias->TypeDict<bool>::dict();
+    auto iter = dict.find(key);
+    if (iter != dict.end()) {
+      if (iter->second.IsSet()) return false;
```

Comment:
I'm not sure I understand how it works.  Where does the value reset to false after reading it the first time?

Reply:
IIRC (it was some time ago) `IsSet()` return value depends on whether the last action was a `Set()` or a `Get()`. Any suggestions for a better implementation are appreciated.